web/src/views/HomeView/HomeView.vue

```diff
@@ -50,18 +50,20 @@
       <StatsBar />
     </v-row>
     <v-row>
-      <Search />
+      <div class="full-width grey darken-2 pa-12">
+        <DandisetSearchField :dense="false" />
+      </div>
     </v-row>
```

Comment:
This can be accomplished without extra css:
```
    <v-row no-gutters>
      <v-col
        class="grey darken-2 pa-12"
        cols="12"
      >
        <DandisetSearchField :dense="false" />
      </v-col>
    </v-row>
```

Which I've tested locally and looks the same. Obviously if you were to do this you'd also have to remove the css below.